packit_service/worker/reporting.py

```diff
@@ -300,23 +325,46 @@ def set_status(
 class StatusReporterGithubChecks(StatusReporterGithubStatuses):
     project_with_commit: GithubProject
 
+    @staticmethod
+    def _create_table_content(
+        url: str, links_to_external_services: Optional[Dict[str, str]]
+    ) -> str:
+        table_content = []
+        if url:
+            table_content.append(f"| Dashboard | {url} |\n")
+        if links_to_external_services is not None:
+            table_content += [
+                f"| {name} | {link} |\n"
+                for name, link in links_to_external_services.items()
+            ]
```

Comment:
```suggestion
            table_content += (
                f"| {name} | {link} |\n"
                for name, link in links_to_external_services.items()
            )
```

This is „just“ syntax thing. `[]` constructs list, `()` is a generator (iterator).

In this use-case it shouldn't matter :thinking: I wonder where's the threshold where the construction of list starts to take more time than the generator. (IIRC there were some benchmarks by Hunor on one of the PRs before, but not sure if it was really this or just related to this.)

[link to SO with benchmark, both time and space](https://stackoverflow.com/a/62709748)

_Conclusion_: Shouldn't make much of a difference for up to 5 links.